src/styles/components/_specification-block.scss

```diff
@@ -18,31 +18,42 @@ $breakpoint-tablet: 820px;
 
   ul {
     list-style: url("../../../public/images/teal-decorative-bullet.svg");
+
+    @media (prefers-color-scheme: dark) {
+      list-style: url("../../../public/images/pink-decorative-bullet.svg");
+    }
   }
 
   code {
-    color: map.get($colors, "blue");
+    color: var(--code-color);
   }
 
   a {
     text-decoration: underline;
-    color: map.get($colors, "black");
-    background: linear-gradient(to left, rgb(0 0 0 / 0%) 50%, map.get($colors, accent-teal) 50%);
-    background-size: 201% 100%;
-    background-position: right bottom;
-    transition: ease-in-out all 250ms;
+    color: var(--foreground-color);
+    outline: none;
```

Comment:
Same with the `outline: none` here as I mentioned above :)

Reply:
Does this work better now? https://github.com/sparkbox/accessible-components/pull/76/commits/3ead839f338f2b6e1faeab0fc8521b3f6fa06d12